src/main/java/com/jcabi/w3c/Validator.java

```diff
@@ -65,6 +65,11 @@
  * from it, but it will be valid.
  *
  * <p>Objects of this interface should be immutable and thread-safe.
+
+ * <p>Please <b>don't use</b> URL as argument of
```

Comment:
Please move this comment to `validate` method, and maybe change it slightly  to something like: This method  expects content of a document which should be validated, not URL to the document.